src/drivers/net/xen_net/netfront.c

```diff
@@ -235,7 +301,7 @@ void network_rx(struct netfront_dev *dev)
         netif_rx_request_t *req = RING_GET_REQUEST(&dev->rx, req_prod + i);
         struct net_buffer* buf = &dev->rx_buffers[id];
         void* page = buf->page;
-
+//TODO virt->mfn
         /* We are sure to have free gnttab entries since they got released above */
         buf->gref = req->gref = gnttab_grant_access(dev->dom,
```

Comment:
Pages should be granted once in initialization. This is fine to receive one-two packets, but need to be changed later